catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -146,7 +146,7 @@ public UpdateResponse update(UpdateRequest updateRequest)
             updateRequest = setDefaultValues(updateRequest);
 
             updateRequest = populateMetacards(updateRequest);
-            updateRequest = processPrePolicyPlugins(updateRequest);
+            updateRequest = processPolicySetupPlugins(updateRequest);
```

Comment:
Nope. Neither is there `processPostCreate` since security doesn't occur after those operations. If someone was allowed to create/update, they should be allowed to see the result. According to @pklinef we should mirror the API with respect to the policy/access layers.